bittide-instances/tests/Wishbone/Axi.hs

```diff
@@ -0,0 +1,132 @@
+-- SPDX-FileCopyrightText: 2022-2024 Google LLC
+--
+-- SPDX-License-Identifier: Apache-2.0
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE NumericUnderscores #-}
+{-# OPTIONS_GHC -fplugin=Protocols.Plugin #-}
+{-# OPTIONS_GHC -fconstraint-solver-iterations=0 #-}
```

Comment:
Don't do this please. If there is a bug in the type checker this can cause it to go into an infinite loop and now you suddenly have a problem whose origin is hard to detect.